### DISCUSSION TITLE
Can't open file on SD

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#147] [Need help]
I put many different ROM file on my SD, 
with some one I have this message: "Can't open file on SD"
Why ?
Long name file ?
_Wonder Project J - Kikai no Shounen Pino (J) [T+Fre1.1_GenerationIX].smc_
Or some Characters are not supported ? 
like "]" "+"   ???

**Comment by sanni** [comment]
File name length currently is max 32 chars and file name + folder path together max 64 chars. It's a limit set in place to reduce RAM usage on the Arduino Mega chip.
